Update error message for insufficient security deposit

Sharqiewicz opened the pull request:
### What:

Change Spacewalk security deposit error
Change Spacewalk tokenSymbol

Closes: #417

ebma reviewed (commented):
The error message looks good but I noticed that for this particular error, the 'Bridge' button does not show as disabled while for any other error regarding the schema, it disables. Any ideas @Sharqiewicz?

<img width="602" alt="image" src="https://github.com/pendulum-chain/portal/assets/6690623/ff420157-af7a-4266-b9fb-70c5a80eb1d4">

Sharqiewicz commented:
> The error message looks good but I noticed that for this particular error, the 'Bridge' button does not show as disabled while for any other error regarding the schema, it disables. Any ideas @Sharqiewicz?
> 
> <img alt="image" width="602" src="https://private-user-images.githubusercontent.com/6690623/330152992-ff420157-af7a-4266-b9fb-70c5a80eb1d4.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU4NDk4NzAsIm5iZiI6MTcxNTg0OTU3MCwicGF0aCI6Ii82NjkwNjIzLzMzMDE1Mjk5Mi1mZjQyMDE1Ny1hZjdhLTQyNjYtYjlmYi03MGM1YTgwZWIxZDQucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDUxNiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA1MTZUMDg1MjUwWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NGRkY2FjZjJkZGQwMzY2N2ViOWQ2ZGFlZTJkMGZkZDc4ZTMyZWQ0ZTRlMzkyNjE1YTA1MGMzMGRhMTc2Zjg2YyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.Dmp9BvsKHQRvkT7XzeyO0pt-0Xtok_qTxKc5TPab9_g">

✅ Very good catch @ebma 
The button was implemented with `disabled={!!formState.errors.amount}` and it is not covering all of the errors that can appear in the form. I changed it to `disabled={!isEmpty(formState.errors}` and also implemented to trigger validation for every onBlur event.

ebma commented on src/pages/bridge/Issue/index.tsx:
```diff
@@ -158,8 +172,10 @@ function Issue(props: IssueProps): JSX.Element {
+    trigger('securityDeposit');
+  }, [amount, issueGriefingCollateral, setValue, trigger]);
 
+  console.log('formState.errors', formState.errors);
```
Nit: remove

ebma commented on src/pages/bridge/Issue/index.tsx:
```diff
@@ -53,8 +66,9 @@ function Issue(props: IssueProps): JSX.Element {
-    resolver: yupResolver(getIssueValidationSchema(maxIssuable, parseFloat(balance || '0.0'))),
+  const { handleSubmit, watch, register, formState, setValue, trigger } = useForm<IssueFormValues>({
+    resolver: yupResolver(getIssueValidationSchema(maxIssuable, parseFloat(balance || '0.0'), tokenSymbol)),
+    mode: 'onBlur',
```
I think `onBlur` is not often enough. Let's trigger the validation as often as possible, ie every time the input changes. Otherwise, I see errors although they should be gone already.

<img width="1671" alt="image" src="https://github.com/pendulum-chain/portal/assets/6690623/25eb3002-8013-4ab1-b9ba-b2e91c1d9288">
